src/components/grid-aware/StatsBlock/StatsBlock.tsx

```diff
@@ -19,17 +19,69 @@ const StatCard = ({ number, secondaryNumber, statement }: StatCardProps) => (
   </div>
 );
 
+type AxisProp = {
+  x: string | number;
+  y: number;
```

Comment:
x and y might not be the best names here because in 2D graphs, the overwhelming convention is for x to be the horizontal dimension and y to be the vertical dimension. In your code, it looks like y is the "stat" that affects the bar size and "x" is the "label" that describes what the bar represents but not its size.

What if you named this type `Bar`, and gave it "label" and "statistic" properties? I'm not sure if "label" and "statistic" are the best names, so please feel free to come up with other ideas, but I think they are at least better than "x" and "y", especially when "x" and "y" might describe the opposite axes that people might expect from a 2D coordinate system.